### PR TITLE
KETTLE-64: Added markdown linting to standard grunt lint checks...

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [KETTLE-64](https://issues.fluidproject.org/browse/KETTLE-64) for details.

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
@amb26, I assume you'd want to pick this one up?

File: package.json [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
-    ],
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/fluid-project/kettle"
-    },
-    "main": "./kettle.js",
-    "engines": {
-        "node": ">=4.5.0"
-    }
+  "name": "kettle",
```
Comment (amb26) [inline package.json]: Please return to 4-spacing here, thanks!

Comment (the-t-in-rtf) [reply package.json]: Fixed.

**Comment by the-t-in-rtf** [comment]
Thanks, @amb26.  My suggestion would be that we explore markdown linting in our shared work for a bit and put in the work to make a common config et cetera once we agree it's of sufficient usefulness.

**Pull request merged**